src/tsk_fs_meta.rs

```diff
@@ -5,7 +5,66 @@ use crate::{
     tsk_fs_dir::TskFsDir,
     bindings as tsk
 };
+use std::fmt::{Display, Formatter, Result as FmtReasult};
+use std::convert::From;
 
+#[derive(Debug)]
+pub enum TskFsMetaFlag {
+    Allocated = 1, 	    //Metadata structure is currently in an allocated state.
+    Unallocated = 2, 	//Metadata structure is currently in an unallocated state.
+    Used = 4, 	        //Metadata structure has been allocated at least once.
+    Unused = 8, 	    //Metadata structure has never been allocated.
+    Compressed = 16,    //The file contents are compressed.
+    Orphan = 32, 	    //Return only metadata structures that have no file name pointing to the (inode_walk flag only)
+    None
+}
+
+impl Display for TskFsMetaFlag{
+    fn fmt(&self, f: &mut Formatter<'_>) -> FmtReasult {
+        write!(f, "{:?}", self)
+    }
+}
+
+#[derive(Debug)]
+pub enum TskFsMetaType {
```

Comment:
Again, not sure if I want to create an enum for something that is auto generated with the bindings. If anything, maybe all we want to do is implement `to_string` for `TSK_FS_META_FLAG_ENUM`, `TSK_FS_META_TYPE_ENUM`, etc.

Reply:
I used `rustified_enum` function from bindgen to create rust enums instead of consts and implemented that.